### DISCUSSION TITLE
IBM Notifications --dropdown

### DISCUSSION THREAD

**Discussion by lukecater** [IBM/mac-ibm-notifications#79] [General]
Hello, I'm trying to use the dropdown feature so my clients in choose which printer they wish to install.
Is there a way for the results to show the name selected as suppose to the number in the list?

**Comment by SMartorelli** [comment]
Hi @lukecater we print out on purpose the index of the array of choices so that you can use it to select the right element on  the ordered source array in the script. It's a more generic way to collect this data in order to be able to provide common approaches to different needs.

**Reply by lukecater** [reply]
Thank you very much.

**Comment by SMartorelli** [comment]
PS: Since this is not to be considered a bug/issue/feature request I'm going to convert it in a discussion.

**Comment by kryptonit1** [comment]
It would be really useful to have the possibility to output the name/value of the list item that the user selected in the dropdown. Especially when the amount of items in the dropdown are generated dynamically and it's not known what items that will be in the dropdown and in which order.

**Reply by SMartorelli** [reply]
Hi @kryptonit1 I suggest to you to save the array of value you pass to IBM Notifier in an array in order to be able to access them by index. As I said to Luke this is the best way we can do it since provide a generic method to interact with an array of elements not considering their value.

**Reply by kryptonit1** [reply]
Ok, thank you for your quick reply! 😊
